aws/sdk-codegen/src/main/kotlin/software/amazon/smithy/rustsdk/AwsEndpointDecorator.kt

```diff
@@ -84,6 +84,7 @@ class EndpointConfigCustomization(private val codegenContext: CodegenContext, pr
             ServiceConfig.BuilderImpl ->
                 rust(
                     """
+                    /// Sets the endpoint resolver to use when making requests.
```

Comment:
can you add a TODO to include an example of using a static endpoint here?
```suggestion
                    // TODO(docs): include an example of using a static endpoint
                    /// Sets the endpoint resolver to use when making requests.
```